### PR TITLE
Color changes

### LINKED ISSUE #4983: Use colors in our UIs in a more meaningful way
#4810 replaced the main orange color with blue in pre-release versions and nobody seems to agree on what should stay orange and what should become blue. My suggestion is to improve the way different colors are used in general:
* Icons and headers would be orange in the stable version and blue in the development version.
image
image
* Orange would indicate elements that were selected, focused, or enabled by the user:
  * tabs and categories:
  image
  image
  * buttons and inputs:
  image
  image
  image
  * `"type": "select"` settings:
  image
  * addon and setting switches:
  image
  * if orange indicates something important or dangerous, it would be changed to yellow:
  image
  image
* Blue would be used for other elements that need to stand out:
  * the main button in a button row:
  image
  * currently opened game in cloud games list:
  image
  * links (always or when hovered):
  image
  image

Badges could also be improved - the amount of different colors makes them look a bit messy.
* I don't think "Forums", "For website" and "For editor" are important enough to have unique colors. Something like this would look fine:
image
* Changing "Beta" to yellow would allow orange to be used for updates, making the additional purple color unnecessary:
image

### PR DESCRIPTION
**Pull request by mxmou**
Resolves #4983
Closes #4952

Also replaces some icons with inline SVGs for easier styling. Marking as draft for discussion.

### CODE REVIEW

File: webpages/common/components/inline-icon.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    viewBox="0 0 24 24"
+    fill="currentcolor"
+    style="transform: rotate(360deg)"
+  >
+    <path
+      v-if="icon === 'cloud'"
+      d="M18.42 9.21a7 7 0 0 0-13.36 1.9A4 4 0 0 0 6 19h11a5 5 0 0 0 1.42-9.79zM17 17H6a2 2 0 0 1 0-4 1 1 0 0 0 1-1 5 5 0 0 1 9.73-1.61 1 1 0 0 0 .78.66A3 3 0 0 1 17 17z"
+    />
+    <path
+      v-if="icon === 'evenlope'"
```
Comment (TheColaber) [inline webpages/common/components/inline-icon.html:16]: You spelled envelope wrong here and here

https://github.com/ScratchAddons/ScratchAddons/blob/ba8c3357b28a99bc39c1f20fe58d59b21123f847/addons/scratch-messaging/addon.json#L13

**Comment by WorldLanguages** [comment]
Can you send screenshots of the visual changes?

Or is the discussion about the creation of an `<inline-icon>` component? We should be careful and check if settings page performance is affected by that. Vue will probably need to handle many more JS objects in memory and there may be more DOM nodes.

**Comment by mxmou** [comment]
> Can you send screenshots of the visual changes?

image
image
image
image
image
image

**Comment by Hans5958** [comment]
This will require changes on the docs, specifically on [the Design page](https://scratchaddons.com/docs/reference/design).

**Comment by mxmou** [comment]
This has been open for almost a year and no progress has been made. I'll probably split it into smaller PRs.

**Pull request closed without merging**